src/navigation/TabBarIcon.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Entypo } from '@expo/vector-icons';
+
```

Comment:
Been feeling the react-native-elements way since it's easy to mix and match from different icon sets. In this case, you'd do something like `<Icon type="entypo" name="megaphone"/>`. Only gotcha is there's a separate style prop for iconStyle and containerStyle.

https://react-native-training.github.io/react-native-elements/docs/icon.html

I like looking through the huge list of icons. Nice to compare and contrast options.
https://oblador.github.io/react-native-vector-icons/